### DISCUSSION TITLE
Making recurring transactions repeat after a set amount of days

### DISCUSSION THREAD

**Discussion by YosypC** [firefly-iii/firefly-iii#5934] [Firefly III] [answered]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)
- [X] My request is not listed as [a very good idea, but unfortunately...](https://docs.firefly-iii.org/firefly-iii/about-firefly-iii/what-its-not/)
- [X] I've used [the search](https://github.com/firefly-iii/firefly-iii/issues?q=is%3Aissue) and this has not been discussed before.

### Description

There are some bills which repeats, at least in my case (Italy), after a set amount of days, for example 28, so not a full month.
I'd be cool if Firefly III allowed recurring transactions to occur after a predefined day count, instead of "X of every month".

### Solution

In "type of repetition", having the option to select "After amount of days" and then having an input box in which you type it.

**Comment by DunklerPhoenix** [comment]
You can set it to weekly and the skip value to 3. Then it will be executed every 4 Weeks (28 days). You can check it with the calendar tool

**Comment by YosypC** [comment]
I was aware of the skip functionality although I didn't really go deep in its usability. In this case, it's a partial solution since 28 coincides with the skip value applied with weeks; had it been slightly more or less, it would have been practically impossible to set, right?

**Comment by DunklerPhoenix** [comment, marked as answer]
Then you set it to a daily interval and set the skip value for a daily base.

e.g. daily + skip 14 = every 15th day

**Comment by YosypC** [comment]
I'm officially dumb. Thank you for the solution.
